File: Aplicacion Windows/TFG_Windows/TFG/BD/MedicationDTO.cs
using System;
using TFG.Enumerados;

namespace TFG.BD
{
    public class MedicationDTO
    {
        public long id {  get; set; }
        public string nombre { get; set; }
        public string descripcion { get; set; }
        public long usuario_id { get; set; }  // Solo el ID del usuario
        public Dias dias { get; set; }
        public Estado status { get; set; }
        public int comprimidos { get; set; }
        public int veces_al_dia { get; set; }
        public string horas { get; set; }
    }

}

File: Aplicacion Movil/MedMonitor/app/src/main/java/Models/MedicationDTO.kt
package Models

import Enums.Dia
import Enums.Estado
import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class MedicationDTO(
    @SerializedName("usuario_id") val usuario_id: Long,
    @SerializedName("nombre") val nombre: String,
    @SerializedName("descripcion") val descripcion: String,
    @SerializedName("dias") val dias: Dia,
    @SerializedName("status") val status: Estado,
    @SerializedName("comprimidos") val comprimidos: Int,
    @SerializedName("veces_al_dia") val veces_al_dia: Int,
    @SerializedName("horas") val horas: String,
) : Serializable
